modules/apps/cookies/cookies-impl/src/main/java/com/liferay/cookies/internal/configuration/helper/CookiesConfigurationHelperImpl.java

```diff
@@ -23,91 +23,66 @@
 import com.liferay.portal.kernel.module.configuration.ConfigurationProvider;
 import com.liferay.portal.kernel.service.LayoutSetLocalService;
 import com.liferay.portal.kernel.theme.ThemeDisplay;
-import com.liferay.portal.kernel.util.WebKeys;
+
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 
-import javax.servlet.http.HttpServletRequest;
-
 /**
  * @author Daniel Sanz
  */
 @Component(immediate = true, service = CookiesConfigurationHelper.class)
 public class CookiesConfigurationHelperImpl
 	implements CookiesConfigurationHelper {
 
-  @Override
-  public CookiesBannerConfiguration getCookiesBannerConfiguration(
-    ThemeDisplay themeDisplay) throws Exception {
-
-    LayoutSet layoutSet = _LayoutSetLocalService.fetchLayoutSet(
-      themeDisplay.getServerName());
-
-    if (layoutSet != null) {
-      Group group = layoutSet.getGroup();
-
-      return _configurationProvider.getGroupConfiguration(
-        CookiesBannerConfiguration.class,
-        group.getGroupId());
-    }
-
-    return _configurationProvider.getCompanyConfiguration(
-      CookiesBannerConfiguration.class,
-      themeDisplay.getCompanyId());
-  }
-
-  @Override
-  public CookiesConsentConfiguration getCookiesConsentConfiguration(
-    ThemeDisplay themeDisplay) throws Exception {
-
-    LayoutSet layoutSet = _LayoutSetLocalService.fetchLayoutSet(
-      themeDisplay.getServerName());
+	@Override
+	public CookiesBannerConfiguration getCookiesBannerConfiguration(
+			ThemeDisplay themeDisplay)
+		throws Exception {
 
-    if (layoutSet != null) {
-      Group group = layoutSet.getGroup();
+		return _getCookiesConfiguration(
+			CookiesBannerConfiguration.class, themeDisplay);
+	}
```

Comment:
This should have been made in https://github.com/liferay-frontend/liferay-portal/pull/2846/commits/32c5d9befa99402eb5db2f90ac576a36917e1426